components/breadcrumbs/index.js

```diff
@@ -0,0 +1,25 @@
+import Link from 'next/link';
+import styles from './breadcrumbs.module.css';
+
+const Breadcrumbs = ({ links }) => {
+  return (
+    <div>
+      {links.map((link, index) => {
+        return (
+          <>
+            <Link key={link} href={`/${link !== 'home' ? link : ''}`}>
+              <p className={styles.links}>
+                {link}
+                {index !== links.length - 1 && (
+                  <>&nbsp;&nbsp;&gt;&nbsp;&nbsp;</>
+                )}
+              </p>
+            </Link>
+          </>
```

Comment:
1. Is the React Fragment needed here ?
2. can this be made into its own component to which we pass the prop `link` and `index`.